cmd/api-linter/integration_test.go

```diff
@@ -96,6 +96,19 @@ func TestRules_DisabledByFileComments(t *testing.T) {
 	}
 }
 
+func TestRules_DisabledByFileCommentsAndIgnoreCommentDisablesTrue(t *testing.T) {
+	for _, test := range testCases {
+		t.Run(test.testName, func(t *testing.T) {
+			disableInFile := fmt.Sprintf("// (-- api-linter: %s=disabled --)", test.rule)
+			proto := disableInFile + "\n" + test.proto
+			_, result := runLinterWithFailureStatus(t, proto, "", []string{"--ignore-comment-disables"})
+			if !strings.Contains(result, test.rule) {
+				t.Errorf("Rule %q should be enabled despite a disable by file comments", test.rule)
+			}
+		})
+	}
+}
+
```

Comment:
Please include a negative test case where you don't pass the flag and expect the rule to be disabled.

Reply:
there already is a negative case that doesn't pass in the flag (TestRules_DisabledByFileComment). are you suggesting I should explicitly test for `--ignore-test-disables=false`?

Reply:
No that's fine.

Reply:
did it anyway :)